CHANGES.rst

```diff
@@ -179,6 +179,10 @@ Bug fixes
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- ``astropy.io.ascii``
+
+  - Update some function docstrings to the Numpy format [#3352]
+
```

Comment:
Ok, I've removed the entry since it isn't necessary. I wasn't originally sure whether it was necessary for such a small change, I only added to be safe after reading the workflow guidelines.